browser/resources/settings/email_aliases_page/content/email_aliases_list.tsx

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { Alias, MappingService, MAX_ALIASES, ViewState } from './types'
+import Card from './styles/Card'
+import Col from './styles/Col'
+import Row from './styles/Row'
+import styled from 'styled-components'
+import Icon from '@brave/leo/react/icon'
+import { color } from '@brave/leo/tokens/css/variables'
+import Tooltip from '@brave/leo/react/tooltip'
+import { getLocale } from '$web-common/locale'
+import formatMessage from '$web-common/formatMessage'
+import Button from '@brave/leo/react/button'
+import ButtonMenu from '@brave/leo/react/buttonMenu'
+
+const AliasItemRow = styled(Row)`
+  margin: 0px;
+  font-size: 125%;
+  padding: 18px 0px 18px 25px;
+  border-top: ${color.divider.subtle} 1px solid;
+  justify-content: space-between;
+`
+
+const AliasAnnotation = styled.div`
+  font-size: 80%;
+  font-weight: 400;
+  padding-top: 0.25em;
+  color: ${color.neutralVariant[50]};
+`
+
+const AliasControls = styled(Row)`
+  height: 1.5em;
+  user-select: none;
+  align-items: top;
+`
+
+const AliasListIntro = styled(Row)`
+  margin-bottom: 20px;
+  justify-content: space-between;
+  & leo-button {
+    flex-grow: 0;
+  }
+`
+
+const EmailContainer = styled.div`
+  cursor: pointer;
+`
+
+const CopyButtonWrapper = styled.div`
+  cursor: pointer;
+  color: ${color.neutralVariant[60]};
+  &:hover {
+    background-color: ${color.desktopbrowser.toolbar.button.hover};
+  }
+  &:active {
+    background-color: ${color.desktopbrowser.toolbar.button.active};
+  }
+  padding: 0.25em;
+  border-radius: 0.5em;
+`
+
+const MenuButton = styled(Button)`
+  --leo-button-padding: 0;
+  flex-grow: 0;
+  width: 1.5em;
```

Comment:
setting a width here is probably not what was intended - it results in this odd looking hover:
![image](https://github.com/user-attachments/assets/43f128f5-a11c-48b0-9bb7-697e884a33db)

Its generally better to just trust the button to size itself correctly

Reply:
fixed